pkgs/top-level/python-packages.nix

```diff
@@ -15567,6 +15578,27 @@ in {
     };
   };
 
+  pipenv = buildPythonPackage rec {
+    name = "pipenv-8.2.7";
+
+    src = pkgs.fetchurl {
```

Comment:
we have a `fetchPypi` function for that.

Reply:
ok, changed